src/Http/SymfonyRequestBridge.php

```diff
@@ -2,197 +2,27 @@
 
 namespace CacheWerk\BrefLaravelBridge\Http;
 
-use RuntimeException;
-
 use Bref\Context\Context;
 use Bref\Event\Http\HttpRequestEvent;
-
-use Riverline\MultiPartParser\StreamedPart;
+use Bref\Event\Http\Psr7Bridge;
+use Symfony\Bridge\PsrHttpMessage\Factory\HttpFoundationFactory;
 use Symfony\Component\HttpFoundation\Request;
-use Symfony\Component\HttpFoundation\File\UploadedFile;
 
 class SymfonyRequestBridge
 {
     /**
-     * Convert Bref HTTP request to Symfony request.
+     * Convert Bref HTTP event to Symfony request.
      *
      * @param  \Bref\Event\Http\HttpRequestEvent  $event
      * @param  \Bref\Context\Context  $context
      * @return \Symfony\Component\HttpFoundation\Request
      */
     public static function convertRequest(HttpRequestEvent $event, Context $context): Request
     {
-        $headers = $event->getHeaders();
-
-        [$user, $password] = self::parseBasicAuthorization($headers);
-
-        $server = array_filter([
-            'AUTH_TYPE' => $headers['auth-type'] ?? null,
-            'CONTENT_LENGTH' => $headers['content-length'][0] ?? null,
-            'CONTENT_TYPE' => $event->getContentType(),
-            'DOCUMENT_ROOT' => getcwd(),
-            'GATEWAY_INTERFACE' => 'FastCGI/1.0',
-            'QUERY_STRING' => $event->getQueryString(),
-            'REQUEST_METHOD' => $event->getMethod(),
-            'SCRIPT_FILENAME' => $_SERVER['_HANDLER'] ?? $_SERVER['SCRIPT_FILENAME'],
-            'SERVER_ADDR' => '127.0.0.1',
-            'SERVER_NAME' => $event->getServerName(),
-            'SERVER_PORT' => $event->getServerPort(),
-            'SERVER_PROTOCOL' => $event->getProtocol(),
-            'PATH_INFO' => $event->getPath(),
-            'REMOTE_PORT' => $event->getRemotePort(),
-            'REQUEST_TIME' => time(),
-            'REQUEST_TIME_FLOAT' => microtime(true),
-            'REQUEST_URI' => $event->getUri(),
-            'REMOTE_ADDR' => '127.0.0.1',
-            'LAMBDA_INVOCATION_CONTEXT' => json_encode($context),
-            'LAMBDA_REQUEST_CONTEXT' => json_encode($event->getRequestContext()),
-            'HTTP_X_SOURCE_IP' => $event->getSourceIp(),
-            'HTTP_X_REQUEST_ID' => $context->getAwsRequestId(),
-            'PHP_AUTH_USER' => $user,
-            'PHP_AUTH_PW' => $password,
```

Comment:
Are these handled/supported in Bref's Psr7Bridge?

Reply:
Added to PR in bref repo